themes/sublime-monokai-color-theme.json

```diff
@@ -552,6 +552,45 @@
 				"foreground": "#66D9EF"
 			}
 		},
+		{
+			"name": "Namespace name",
+			"scope": "entity.name.namespace.cpp",
+			"settings": {
+				"foreground": "#A6E22E",
+			}
+		},
+		{
+			"name": "Namespace keyword",
+			"scope": "storage.type.namespace.definition.cpp",
+			"settings": {
+				"foreground": "#F92672",
+				"fontStyle": ""
+			}
+		},
+		{
+			"name": "Access modifier keyword",
+			"scope": "storage.type.modifier.access",
+			"settings": {
+					"foreground": "#F92672",
+					"fontStyle": ""
+			}
+		},
+		{
+			"name": "Access modifier colon",
+			"scope": "punctuation.separator.colon.access.control.cpp",
+			"settings": {
+				"foreground": "#F8F8F2",
+			}
+		},
+		{
+			// Not perfect, but the best we can do without updating C++ language support
```

Comment:
indeed, but in practice it worked just fine isn't it? At least from the two screenshots you sent they look identical, which is the intention of the theme 😁